dll/cpl/sysdm/lang/fr-FR.rc

```diff
@@ -330,11 +330,11 @@ BEGIN
     IDS_USERPROFILE_ACCOUNT_UNKNOWN "Account Unknown"
     IDS_USERPROFILE_ACCOUNT_DELETED "Account Deleted"
     IDS_USERPROFILE_TYPE_TEXT "When %s logs onto this computer, should the operating system use the roaming profile or just the locally cached copy of the roaming profile."
-    IDS_MESSAGEBOXTITLE "System control panel applet"
+    IDS_MESSAGEBOXTITLE "Application Système du Panneau de configuration"
     IDS_WARNINITIALSIZE "Entrez une valeur numérique pour la taille initiale du fichier d'échange."
     IDS_WARNMAXIMUMSIZE "Entrez une valeur numérique pour la taille maximale du fichier d'échange."
-    IDS_WARNINITIALRANGE "La taille initiale du fichier d'échange ne peut pas être inférieure à 16 Mo and ne doit pas dépasser l'espace libre total du disque."
-    IDS_WARNMAXIMUMRANGE "La taille maximale du fichier d'échange ne peut pas être inférieure à sa taille initiale, et ne peut pas être supérieure à 4095 Mo et ne peut pas dépasser l'espace libre total du disque."
+    IDS_WARNINITIALRANGE "La taille initiale du fichier d'échange doit être comprise entre 2 Mo et 4096 Mo, et ne peut pas excéder la quantité d'espace libre sur le lecteur que vous avez sélectionné."
```

Comment:
I wonder how this change slipped into master without being synced with other translations, even with default english one. https://github.com/reactos/reactos/blob/756152936b1097a3ee228b1d3c5e364a7c1aeec4/dll/cpl/sysdm/lang/en-US.rc#L336

Reply:
I am preparing a complementary PR

Reply:
https://github.com/reactos/reactos/pull/2706 : To be approved and merged in fast lane if you consider it is a regression to be fixed.